vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.h

```diff
@@ -454,7 +445,7 @@ class LedgerImpl : public ledger::Ledger {
 
   void SetStringState(const std::string& name, const std::string& value);
 
-  std::string GetStringState(const std::string& name) const;
+  virtual std::string GetStringState(const std::string& name) const;
```

Comment:
Curious: any particular reason this was made virtual?

Reply:
it was changed to virtual so that can be mocked in unit tests